todo_app/listpage/views.py

```diff
@@ -0,0 +1,28 @@
+from django.shortcuts import render
+from objectivepage.models import Objective
+from targetpage.models import Target
+
+# Create your views here.
+def display_list(request):
+    objectives = Objective.objects.filter(
+        user=request.user
+    ).order_by('id').reverse()
```

Comment:
SHOULD: reverseせず、 `-id` を使いましょう。

また、 idではなくpkを使いましょう（idはカラム名ですが、pkならどんなカラム名でもDjangoが吸収してくれます）

```suggestion
    ).order_by('-pk')
```